pages/sign-up.tsx

```diff
@@ -161,6 +187,8 @@ function SignUpPage(props: any) {
               {
                 password: state.password,
                 username: state.username,
+                // add confirm password
```

Comment:
nit: we don't need this comment, it is obvious in the naming.

Reply:
comment removed